### ISSUE TITLE
Regression: ngx_pagespeed breaking CSS on pages

### ISSUE THREAD

**Issue by mqudsi** [pagespeed/ngx_pagespeed#924]
I'm not sure what info to provide for this one, but after upgrading ngx_pagespeed last week (same time as my other bug reports), I sporadically have problems with broken CSS on the site. I have to clear the pagespeed cache for it to be regenerated correctly.

HTML for broken page: http://pastebin.com/bSUYH3AJ

HTML for correct page: http://pastebin.com/k8rLgNET

I have attached pictures of the broken and correct site as well.

![good](https://cloud.githubusercontent.com/assets/606923/6316495/2b75fd3c-b9ee-11e4-84ef-ec8ed144f24c.png)
![bad](https://cloud.githubusercontent.com/assets/606923/6316496/2b76cf8c-b9ee-11e4-97ff-d6ac5979e299.png)

**Comment by crowell** [comment]
What version of pagespeed were you using, and what did you upgrade to?

**Comment by mqudsi** [comment]
I'm embarrassed to say I do not know what version I was running before.
I'm currently running off of trunk.

**Comment by jeffkaufman** [comment]
That looks like a pageload where css loads entirely.

You said that it gets into this state, and then it's bad until you clear the cache?  Next time this happens, could you run webpagetest.org against your site and send us a link?

Also, it would be good to open the network inspector in the browser and see which urls are failing to load.

What pagespeed options do you have set?

**Comment by mqudsi** [comment]
No requests fail, and it stays bad until I clear the cache or the cache expires.
I just experienced it and it seems it's due to an error earlier up in the chain, as this appears in the log:

````[Sun, 15 Mar 2015 21:43:00 GMT] [Warning] [30503] CSS parsing error in https://neosmart.net/blog/2015/hello-ireboot-2-0/````

This is the full message_history window at the time: http://pastebin.com/ay8MxsVb

**Comment by mqudsi** [comment]
This is the webpagetest result for the broken page: http://www.webpagetest.org/result/150315_S7_WXJ/
This is the webpagetest for a good page (after clearing cache w/ ngx_pagespeed still enabled): http://www.webpagetest.org/result/150315_M8_X06/

**Comment by jeffkaufman** [comment]
Is this the same bug you later reported as #963?

**Comment by mqudsi** [comment]
No, in this bug the page would remain broken until the cache is flushed or the cache entry expires.

I'm not sure if this bug still happens.
